plugins/flytekit-huggingface/tests/test_huggingface_plugin_sd.py

```diff
@@ -0,0 +1,65 @@
+import datasets
+import flytekitplugins.huggingface  # noqa F401
+import pandas as pd
+
+try:
+    from typing import Annotated
+except ImportError:
+    from typing_extensions import Annotated
+
+from flytekit import kwtypes, task, workflow
+from flytekit.types.structured.structured_dataset import PARQUET, StructuredDataset
+
+subset_schema = Annotated[StructuredDataset, kwtypes(col2=str), PARQUET]
+full_schema = Annotated[StructuredDataset, PARQUET]
+
+
+def test_huggingface_dataset_workflow_subset():
+    @task
+    def generate() -> subset_schema:
+        df = pd.DataFrame({"col1": [1, 3, 2], "col2": list("abc")})
```

Comment:
Could we use `datasets.Dataset` here? we have to test that we can successfully convert the dataset to parquet.